src/schemas/job/JobResponse.yaml

```diff
@@ -153,3 +153,6 @@ allOf:
                   For example: `* * * * *` represent the cron to launch the job every minute.  
                   See https://crontab.guru/ to WISIWIG interface.  
                   Timezone is UT
+      auto_deploy:
+        type: boolean
+        description: Specify if the job would be automatically updated after receiving a new image or a new commit according to the source type.
```

Comment:
Specify if the job will be automatically updated after receiving a new image tag or a new commit according to the source type. The new image tag shall be communicated via the "Auto Deploy container" endpoint https://api-doc.qovery.com/#tag/Containers/operation/autoDeployContainerEnvironments

Reply:
done